WMFComponents/Sources/WMFComponents/Components/Year in Review/WMFYearInReview.swift

```diff
@@ -80,4 +86,47 @@ public struct WMFYearInReview: View {
         .environment(\.colorScheme, theme.preferredColorScheme)
         .frame(maxHeight: .infinity)
     }
+
+    private var firstSlide: some View {
+        ScrollView {
+            VStack(spacing: 48) {
+                VStack(alignment: .leading, spacing: 16) {
+                    Image("globe", bundle: .module)
+                        .frame(maxWidth: .infinity, alignment: .center)
+                    Text(viewModel.localizedStrings.firstSlideTitle)
+                        .font(Font(WMFFont.for(.boldTitle1)))
+                        .foregroundStyle(Color(uiColor: theme.text))
+                    Text(viewModel.localizedStrings.firstSlideSubtitle)
+                        .font(Font(WMFFont.for(.title3)))
+                        .foregroundStyle(Color(uiColor: theme.text))
+                }
+                VStack {
+                    Button(action: {
```

Comment:
Can you use WMFLargeButton for this instead? We can explain to design that we're trying to use existing button components here for consistency. This is used in our other onboarding view (WMFOnboardingView).

```
WMFLargeButton(configuration: .primary, title: viewModel.localizedStrings.firstSlideCTA) {
                        withAnimation {
                            viewModel.isFirstSlide = false
                        }
                    }
```